lib/alphavantage/validations.rb

```diff
@@ -51,5 +51,14 @@ def validate_mat(moving_average_type)
     def is_integer?(str)
       Integer(str) rescue false
     end
+
+    private
+
+    def validate_from_collection(value:, collection:, type:)
```

Comment:
@kvokka awesome, thanks for this. Wanted to refactor this but didn't have the time 👍🏻